src/utils/RouteUtils.js

```diff
@@ -222,7 +223,33 @@ async function parseRoute(resp: Object, destinationName: string) {
                     try {
                         // TODO refactor
                         // eslint-disable-next-line no-await-in-loop
-                        const snappingResponse = await axios.post(`http://${process.env.MAP_MATCHING || 'ERROR'}:8989/match`, gpx, config);
+
+                        const options = {
+                            method: 'POST',
+                            url: `http://${process.env.MAP_MATCHING || 'ERROR'}:8989/match`,
+                            body: gpx,
+                            headers:
+                                {
+                                    'Content-Type': 'application/xml',
+                                },
+                            qs:
```

Comment:
same thing here, or at least have the opening bracket on the same line as `qs: `, so `qs: {`